CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Updated iOS icons query.
+
+## [1.36.0] - 2019-01-26
```

Comment:
Is it right?

Reply:
Yep. @brunojdo forgot [this](https://github.com/vtex-apps/store/commit/35f0147be80c03218e376557d0b16502338facb2)